packages/client/src/views/SysAdmin/Config/FormConfigWizard.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { IStoreState } from '@client/store'
+import { getScope } from '@client/profile/profileSelectors'
+import { Scope } from '@client/utils/authUtils'
+import { injectIntl, WrappedComponentProps as IntlShapeProps } from 'react-intl'
+import { RouteComponentProps, Redirect } from 'react-router'
+import { HOME } from '@client/navigation/routes'
+import { EventTopBar } from '@opencrvs/components/lib/interface'
+
+import { SettingsBlue } from '@opencrvs/components/lib/icons'
+import {
+  SecondaryButton,
+  SuccessButton
+} from '@opencrvs/components/lib/buttons'
+import { goBack } from 'connected-react-router'
+import styled from '@client/styledComponents'
+import {
+  PageNavigation,
+  TAB_BIRTH,
+  TAB_DEATH
+} from '@client/components/formConfig/PageNavigation'
+import { FormTools } from '@client/components/formConfig/formTools/FormTools'
+import { FormConfigCanvas } from '@client/components/formConfig/FormConfigCanvas'
+import { IForm } from '@client/forms'
+import { getRegisterForm } from '@client/forms/register/declaration-selectors'
+import { goToPageNavigation } from '@client/navigation'
+import { buttonMessages } from '@client/i18n/messages'
+
+export enum EventType {
+  BIRTH = 'birth',
+  DEATH = 'death'
+}
+
+type RouteProps = RouteComponentProps<{
+  event: string
+  section: string
+}>
+
+interface IStateProps {
+  scope: Scope | null
+  registerForm: { [key: string]: IForm }
+  event?: EventType
+  section?: string | undefined
+}
+
+interface IDispatchProps {
+  goBack: typeof goBack
+  goToPageNavigation: typeof goToPageNavigation
+}
+
+const WizardContainer = styled.div`
+  margin-top: 56px;
+  @media (max-width: ${({ theme }) => theme.grid.breakpoints.lg}px) {
+    display: none;
+  }
+`
+
+type IFullProps = IDispatchProps & IStateProps & IntlShapeProps & RouteProps
+
+const topBarActions = (props: IFullProps) => {
+  return [
+    <SettingsBlue onClick={() => {}} />,
+    <SecondaryButton size="small" onClick={() => {}}>
+      {props.intl.formatMessage(buttonMessages.save)}
+    </SecondaryButton>,
+    <SuccessButton size="small" onClick={() => {}}>
+      {props.intl.formatMessage(buttonMessages.publish)}
+    </SuccessButton>
+  ]
+}
+
+function FormConfigWizardComp(props: IFullProps) {
+  if (
+    !props.scope?.includes('natlsysadmin') ||
+    !props.event ||
+    !props.section
+  ) {
+    return <Redirect to={HOME} />
+  }
+
+  return (
+    <>
+      <EventTopBar
+        title={'Birth v0.1'}
+        pageIcon={<></>}
+        topBarActions={topBarActions(props)}
+        goHome={() => props.goBack()}
+      />
+      <WizardContainer>
+        <PageNavigation
+          registerForm={props.registerForm}
+          event={props.event}
+          intl={props.intl}
+          section={props.section}
+          goToPageNavigation={props.goToPageNavigation}
+        />
+        <FormConfigCanvas></FormConfigCanvas>
+        <FormTools intl={props.intl} />
```

Comment:
Instead of defining the layouts inside the component, it would be better to define them here as a wrapper for the component.